README.md

```diff
@@ -116,6 +123,10 @@ $ docker run bitnami/postgresql cat /opt/bitnami/postgresql/.spdx-postgresql.spd
 
 ## Deprecation notes
 
+### 2023-11
+
+* Apache MXNet
```

Comment:
The linter rules seems to be updated and now these lines fail. Do you mind fixing all of them? Thanks!

```
Checking 'README.md'
README.md:43:1 MD004/ul-style Unordered list style [Expected: dash; Actual: asterisk]
README.md:44:1 MD004/ul-style Unordered list style [Expected: dash; Actual: asterisk]
README.md:50:1 MD004/ul-style Unordered list style [Expected: dash; Actual: asterisk]
README.md:65:1 MD004/ul-style Unordered list style [Expected: dash; Actual: asterisk]
README.md:118:1 MD004/ul-style Unordered list style [Expected: dash; Actual: asterisk]
README.md:128:1 MD004/ul-style Unordered list style [Expected: dash; Actual: asterisk]
README.md:132:1 MD004/ul-style Unordered list style [Expected: dash; Actual: asterisk]
README.md:133:1 MD004/ul-style Unordered list style [Expected: dash; Actual: asterisk]
README.md:137:1 MD004/ul-style Unordered list style [Expected: dash; Actual: asterisk]
README.md:141:1 MD004/ul-style Unordered list style [Expected: dash; Actual: asterisk]
README.md:142:1 MD004/ul-style Unordered list style [Expected: dash; Actual: asterisk]
README.md:143:1 MD004/ul-style Unordered list style [Expected: dash; Actual: asterisk]
README.md:144:1 MD004/ul-style Unordered list style [Expected: dash; Actual: asterisk]
```

Reply:
Updated